lineman/app/components/mixins.scss

```diff
@@ -135,9 +134,44 @@ $block-border-color: darken($block-color, 10%);
   } @else if $level == 5 {
     box-shadow: 0 19px 38px rgba(0,0,0,0.30), 0 15px 12px rgba(0,0,0,0.22);
   }
-
 }
 
+@mixin dropdown_arrow {
```

Comment:
Also the class should be dropdown-arrow rather than .arrow  the idea with BEM is that everything is namespaced and nesting is not used because it's too complex and dumb